Makefile.config

```diff
@@ -54,6 +54,11 @@ else
         $(warning Warn: Building without nftables support)
 endif
 
+ifeq ($(call try-cc,$(FEATURE_TEST_NO_DANGLING_POINTER),-Wno-dangling-pointer),true)
```

Comment:
pls add a comment that explains why -Wno-dangling-pointer should be set.

Reply:
done